clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -173,6 +173,13 @@ func (p *probesMeasurement) start(config *measurement.Config) error {
 		return err
 	}
 	if err := p.waitForProbesReady(config); err != nil {
+		if err.Error() == wait.ErrWaitTimeout.Error() {
```

Comment:
What about timing out for all pods? I would expect a full fail then.

current code:

	expectedTargets := p.replicasPerProbe * len(p.config.ProbeLabelValues)
	return prometheus.CheckAllTargetsReady()

my proposition:

	allTargets := p.replicasPerProbe * len(p.config.ProbeLabelValues)
	if fraction > 0 {
		allowedUnhealthyTargets := max(1, fraction * allTargets)
	}

not sure if that fraction should be same for all probe types
what about ping servers and clients - would % work for that as well?

maybe instead of CheckAllTargetsReady, that could be used (not sure if probes are unready or not visible at all in prometheus):

	CheckTargetsReady(k8sClient kubernetes.Interface, selector func(Target) bool, minActiveTargets, minReadyTargets int)